ledger/error/src/main/scala/com/daml/error/ErrorCategory.scala

```diff
@@ -346,6 +346,18 @@ object ErrorCategory {
       )
       with ErrorCategory
 
```

Comment:
I wanted to get rid of below category and just map the grpc code for the specific error categories until we changed the ledger API conformance tests. In that sense, it might be better to never introduce this category in the daml repo and just do the fix right away.

Reply:
Sorry, @daravep , I don't follow. Can you please expand?

Reply:
I see it is used for `InterpreterErrors.LookupErrors.ContractNotFound` and `ConsistencyRejections.DuplicateKey` in canton. But I think the errors of these kinds can originate in multiple places, for instance here:
https://github.com/digital-asset/daml/blob/30f74adc993e41b057cef7a15dd4575971d62181/ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiSubmissionService.scala#L296

Anyhow, I would not want to embark on error removal as part of this already big PR,  it should be done separately.